next.config.js

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
+  webpack: (config) => {
```

Comment:
is this change still necessary?

Reply:
yea no need, since we dont have outer awaits.